packages/core/src/common/debounce.spec.ts

```diff
@@ -0,0 +1,28 @@
+/********************************************************************************
```

Comment:
Since `debounceAsync` is under `promise-util.ts`, the spec file should also be under that name `promise-util.spec.ts`.